### ISSUE TITLE
Path rendering inconsistency between react-native and web

### ISSUE THREAD

**Issue by bobderrico80** [react-native-community/react-native-svg#822]
I have a react-native-svg `Svg` defined here:

```
<Svg height="128" width="106">
	<Rect fill="none" height="128" width="106" />
	<Path d="m11.07 1.2e2 l0.84-9.29 c1.97-18.79 23.34-22.93 41.09-22.93 17.74 0 39.11 4.13 41.08 22.84l0.84 9.38h10.04l-0.93-10.34c-2.15-20.43-20.14-31.66-51.03-31.66s-48.89 11.22-51.05 31.73l-0.91 10.27h10.03m41.93-102.29c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98s18.24-10.31 18.24-23.98c0-9.9-8.52-18.59-18.24-18.59zm0 52.29c-15.96 0-28-14.48-28-33.67 0-15.36 12.82-28.33 28-28.33s28 12.97 28 28.33c0 19.19-12.04 33.67-28 33.67" />
</Svg>
```

This is a port of this original svg element:
```
<svg xmlns="http://www.w3.org/2000/svg" height="128" version="1.1" viewBox="0 0 106 128" width="106">
	<rect fill="none" height="128" width="106" />
	<path d="m11.07 1.2e2l0.84-9.29c1.97-18.79 23.34-22.93 41.09-22.93 17.74 0 39.11 4.13 41.08 22.84l0.84 9.38h10.04l-0.93-10.34c-2.15-20.43-20.14-31.66-51.03-31.66s-48.89 11.22-51.05 31.73l-0.91 10.27h10.03m41.93-102.29c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98s18.24-10.31 18.24-23.98c0-9.9-8.52-18.59-18.24-18.59zm0 52.29c-15.96 0-28-14.48-28-33.67 0-15.36 12.82-28.33 28-28.33s28 12.97 28 28.33c0 19.19-12.04 33.67-28 33.67" />
	</svg>
```

When I view this SVG on the web, I see this:
<img width="128" alt="screen shot 2018-10-23 at 4 10 27 pm" src="https://user-images.githubusercontent.com/7296842/47387774-50b62c00-d6de-11e8-97f0-f645b3c44c84.png">

However, in react-native, it looks like this:
![screen shot 2018-10-23 at 4 10 37 pm](https://user-images.githubusercontent.com/7296842/47387794-5f044800-d6de-11e8-9dc6-c52b6cbb2023.png)

After experimenting with the path definition for a while, I discovered that I can fix the issue by modifying my `Svg` component to this:
```
<Svg height="128" width="106">
	<Rect fill="none" height="128" width="106" />
	<Path d="m11.07 1.2e2 l0.84-9.29 c1.97-18.79 23.34-22.93 41.09-22.93 17.74 0 39.11 4.13 41.08 22.84l0.84 9.38h10.04l-0.93-10.34c-2.15-20.43-20.14-31.66-51.03-31.66s-48.89 11.22-51.05 31.73l-0.91 10.27h10.03m41.93-102.29c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98s18.24-10.31 18.24-23.98c0-9.9-8.52-18.59-18.24-18.59m0 52.29c-15.96 0-28-14.48-28-33.67 0-15.36 12.82-28.33 28-28.33s28 12.97 28 28.33c0 19.19-12.04 33.67-28 33.67" />
</Svg>
```
Here, I removed the `z` ClosePath command before the final `m` command. This has the effect of repositioning the circle in the bottom left that is being cut off into its rightful position overtop of the center 'head' circle.

Note that making the change (removing the `z`) had no adverse effect when viewing the SVG on the web.

Is this inconsistency a result of a bug in the react-native path parser, or is the issue with the SVG being invalid itself (and perhaps the browser is being more forgiving)?

**Comment by msand** [comment]
Published a fix for the spec conformance of the path parser: https://github.com/react-native-community/react-native-svg/commit/9affd59b008500e6704c20458186899fd4fc5b70
Available as v8.0.4

https://www.w3.org/TR/SVG/paths.html#PathDataMovetoCommands
> The "moveto" commands (M or m) must establish a new initial point and a new current point.

**Comment by bobderrico80** [comment]
@msand Upgrading to `^8.0.4` fixes the issue. Thank you for the very prompt turn-around!
